src/pages/CreatePool/index.tsx

```diff
@@ -264,6 +259,9 @@ export default function CreatePool(): React.ReactElement {
             variant="contained"
             size="large"
             fullWidth
+            disabled={
+              poolNameError || poolSymbolError || feeError || parameterError
```

Comment:
This will need to be disabled if one of necessary fields is not filled out as well.

Reply:
Right!. Updated!